src/renderer/Renderer.cpp

```diff
@@ -210,7 +216,7 @@ CRenderer::SRenderFeedback CRenderer::renderLock(const CSessionLockSurface& surf
 
     Debug::log(TRACE, "frame {}", frames);
 
-    feedback.needsFrame = feedback.needsFrame || !asyncResourceGatherer->ready || bga < 1.0;
+    feedback.needsFrame = feedback.needsFrame || !asyncResourceGatherer->ready || bga != 1.0;
```

Comment:
what's the reason behind this

Reply:
I was making a few changes while debugging, forgot to change that, I'll switch it back. Thanks for pointing it out!